src/layers/tile/infinite.rs

```diff
@@ -76,6 +76,18 @@ impl InfiniteTileLayerData {
             })
             .flatten()
     }
+
+    /// Returns an iterator over only the data part of the chunks of this tile layer.
+    ///
+    /// In 99.99% of cases you'll want to use [`Self::chunks()`] instead; Using this method is only
+    /// needed if you *only* require the tile data of the chunks (and no other utilities provided by
+    /// the map-wrapped [`LayerTile`]), and you are in dire need for that extra bit of performance.
+    ///
+    /// This iterator doesn't have any particular order set.
```

Comment:
Maybe remove "set" at the end of the sentence?
```suggestion
    /// This iterator doesn't have any particular order.
```